sdk/src/inflation.rs

```diff
@@ -22,7 +22,7 @@ pub struct Inflation {
     __unused: f64,
 }
 
-const DEFAULT_INITIAL: f64 = 0.15;
+const DEFAULT_INITIAL: f64 = 0.08;
```

Comment:
@ericlwilliams @mvines @CriesofCarrots @t-nelson 
another inflation overlooks..

This is safe without feature gate because this is only referenced when changing bank.inflation; Usually bank.inflation is carried way when not changing it. Also encoded in snapshots.

Reply:
Oof, good catch!